R/use_standalone.R

```diff
@@ -84,20 +87,27 @@ use_standalone <- function(repo_spec, file = NULL, ref = NULL, host = NULL) {
   }
 
   imports <- dependencies$imports
-
-  for (i in seq_len(nrow(imports))) {
-    import <- imports[i, , drop = FALSE]
-
-    if (is.na(import$ver)) {
-      ver <- NULL
-    } else {
-      ver <- import$ver
+  suggests <- dependencies$suggests
+
+  add_packages <- function(x, type = c("Imports", "Suggests")) {
+    type <- arg_match(type)
+
+    for (i in seq_len(nrow(x))) {
+      pkgs <- x[i, , drop = FALSE]
+      if (is.na(pkgs$ver)) {
+        ver <- NULL
+      } else {
+        ver <- pkgs$ver
+      }
+      ui_silence(
+        use_package(pkgs$pkg, min_version = ver, type = type)
```

Comment:
```suggestion
        use_dependency(pkgs$pkg, min_version = ver, type = type, allow_decrease = FALSE)
```

If #2077 gets merged